src/Symfony/Bundle/TwigBundle/DependencyInjection/Compiler/TemplatingPass.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\TwigBundle\DependencyInjection\Compiler;
+
+use Symfony\Component\Config\FileLocator;
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Loader\XmlFileLoader;
+use Symfony\Component\Templating\EngineInterface;
+
+/**
+ * @author Maxime Steinhausser <maxime.steinhausser@gmail.com>
+ */
+class TemplatingPass implements CompilerPassInterface
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function process(ContainerBuilder $container)
+    {
+        if (!interface_exists(EngineInterface::class) || !$container->hasAlias('templating')) {
```

Comment:
I've kept the `interface_exists` check as a safe guard, in case another `templating` alias exists in userland/third party bundle (probably unlikely, but still).

We could also get the framework configuration and process it back to get the `$config['templating']['enabled]` value, but not sure it's worth it.